verification/curator-service/ui/src/components/BulkCaseForm.tsx

```diff
@@ -270,21 +315,60 @@ class BulkCaseForm extends React.Component<
     createAgeRange(c: RawParsedCase): AgeRange {
         let ageRangeStart = c.ageRangeStart;
         let ageRangeEnd = c.ageRangeEnd;
-        if (c.ageRange?.match(/\d*-\d*/)) {
+        if (c.ageRange?.match(/^\d*-\d*$/)) {
             const startEnd = c.ageRange.split('-');
             ageRangeStart = startEnd[0] ? Number(startEnd[0]) : undefined;
             ageRangeEnd = startEnd[1] ? Number(startEnd[1]) : undefined;
         }
         return { start: ageRangeStart, end: ageRangeEnd };
     }
 
+    createSymptoms(c: RawParsedCase): Symptoms | undefined {
+        let status = c.symptomStatus;
+        if (!status && (c.symptoms || c.dateSymptomOnset)) {
+            status = 'Symptomatic';
```

Comment:
This could be `Presymptomatic` too so we shouldn't fill this in for them

Reply:
Fair question -- I'd sort of agreed to go along with the direction in #646 on the basis that `Symptomatic` should be far more prevalent, but let's get Stephen's thoughts.